crates/goose/src/session/storage.rs

```diff
@@ -31,6 +31,12 @@ fn get_home_dir() -> PathBuf {
         .to_path_buf()
 }
 
+fn get_current_working_dir() -> PathBuf {
+    std::env::current_dir()
+        .or_else(|_| Ok::<PathBuf, io::Error>(get_home_dir()))
+        .expect("could not determine the current working directory")
+}
```

Comment:
excellent. I wonder if we should avoid panicking here though - either return a default "", or bubble up?

fn get_current_working_dir() -> Result<PathBuf, io::Error> {
    std::env::current_dir().or_else(|_| Ok(get_home_dir()))
}